{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2021-28378"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "Gitea 1.12.x and 1.13.x before 1.13.4 allows XSS via certain issue data in some situations."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://blog.gitea.io/2021/03/gitea-1.13.4-is-released/",
          "refsource": "MISC",
          "tags": [],
          "url": "https://blog.gitea.io/2021/03/gitea-1.13.4-is-released/"
        },
        {
          "name": "https://github.com/go-gitea/gitea/pull/14898",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/go-gitea/gitea/pull/14898"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2021-03-15T06:15Z",
  "publishedDate": "2021-03-15T06:15Z"
}